styles.css

```diff
@@ -0,0 +1,79 @@
+div.container {
+  width: 60rem;
+  height: 640px;
+}
+
+.row {
+  display: flexboxgrid;
+  flex-direction: column;
+}
+
+.col {
+  flex: 1;
+  margin: 0.1em;
+  padding: 0.8em;
+}
+
+div.difference-header {
+  margin-top: 50px;
+  margin-bottom: 15px;
+}
+
+h1 {
+  font-size: 1.9em;
+}
+
+.difference-images {
+  width: 270px;
+}
+
+.centered {
+  text-align: center;
+}
+
+.bolder-text {
+  font-family: 'ColfaxWebMedium';
+  font-size: 0.97em;
+}
+
+.regular-text {
+  font-family: 'ColfaxRegular';
+  font-size: 0.8em;
+  line-height: 1.5;
+}
+
+.darker-blue-text {
+  color: rgb(0, 51, 85);
+}
+
+.blue-text {
```

Comment:
This is probably supposed to be for `a` tags, feel free to add a CSS rule for `a` and remove this class.